From: Roel Kluin <roel.kluin@gmail.com>
Date: Fri, 18 Dec 2009 20:16:10 -0800
Subject: [PATCH 050/207] broadcom: bcm54xx_shadow_read() errors ignored in
 bcm54xx_adjust_rxrefclk()

commit 5ee6f6a17cfde9c3141e4d57cf88b5cdf638b463 upstream.

If not signed read errors are ignored.

Signed-off-by: Roel Kluin <roel.kluin@gmail.com>
Acked-by: Matt Carlson <mcarlson@broadcom.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
---
 drivers/net/phy/broadcom.c |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/drivers/net/phy/broadcom.c b/drivers/net/phy/broadcom.c
index f433a2e..996ec9e 100644
--- a/drivers/net/phy/broadcom.c
+++ b/drivers/net/phy/broadcom.c
@@ -326,7 +326,8 @@ error:
 
 static void bcm54xx_adjust_rxrefclk(struct phy_device *phydev)
 {
-	u32 val, orig;
+	u32 orig;
+	int val;
 	bool clk125en = true;
 
 	/* Abort if we are using an untested phy. */
-- 
1.7.5.4

